packages/compose/src/hooks/use-resize-observer/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import useResizeAware from 'react-resize-aware';
+
+/**
+ * Hook which allows to listen the resize event of any target element when it changes sizes.
+ * _Note: `useResizeObserver` will report `null` until after first render_
+ *
+ * @return {?Object} Measurements object with properties `width` and `height`.
```

Comment:
I believe it should be an Array. Maybe you could even describe what is inside :)